.github/workflows/pull-request.yml

```diff
@@ -31,6 +31,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     env:
       USE_BAZEL_VERSION: 5.4.0
+    timeout-minutes: 45
```

Comment:
is it really expected to take so long? in ovirt/vsphere it's around 20m?

Reply:
its really the worse case here,it usually takes  from 20 min to 40 min, since its running a real packstack container , with all the core open-stack services, it can take longer sometimes.